### ISSUE TITLE
Compile scss error

### ISSUE THREAD

**Issue by ajcarrillo** [540co/govicons#59]
There is an error in `govicons/scss/govicons.scss` file line 18, instead of use `screen-reader` should be use it `screen_reader`

**Comment by ostranme** [comment]
Good catch, I'll make a fix soon to align the file name with the variable name and push a patched version.

**Comment by ostranme** [comment]
@ajcarrillo - Updated file name `screen-reader.scss` to match sass variable name in `govicons.scss`. New version `1.4.1` has been published.

https://github.com/540co/govicons/releases/tag/1.4.1

Thanks!
